packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/FishjamClientInternal.kt

```diff
@@ -621,23 +630,20 @@ internal class FishjamClientInternal(
     prevTracks = mutableListOf()
   }
 
-  override fun onOfferData(
-    integratedTurnServers: List<OfferData.TurnServer>,
-    tracksTypes: Map<String, Int>
-  ) {
+  override fun onOfferData(tracksTypes: Server.MediaEvent.OfferData.TrackTypes) {
     coroutineScope.launch {
       try {
         val offer =
           peerConnectionManager.getSdpOffer(
-            integratedTurnServers,
             tracksTypes,
             localEndpoint.tracks.values.toList()
           )
 
         rtcEngineCommunication.sdpOffer(
           offer.description,
           localEndpoint.tracks.map { (_, track) -> track.webrtcId() to track.metadata }.toMap(),
-          offer.midToTrackIdMapping
+          offer.midToTrackIdMapping,
+          localEndpoint.tracks.map { (_, track) -> track.webrtcId() to 1500000 }.toMap() // TODO: Update with simulcast
```

Comment:
WDYT about adding link to simulcast project in linear?
```suggestion
          localEndpoint.tracks.map { (_, track) -> track.webrtcId() to 1500000 }.toMap() // TODO(FCE-XXX): Update with simulcast
```